src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaCandidate.php

```diff
@@ -27,39 +27,24 @@
 use Surfnet\Stepup\Identity\Value\NameId;
 
 /**
- * @ORM\Entity(repositoryClass="Surfnet\StepupMiddleware\ApiBundle\Identity\Repository\RaCandidateRepository")
- * @ORM\Table(
- *      indexes={
- *          @ORM\Index(name="idx_ra_candidate_institution", columns={"institution"}),
- *          @ORM\Index(name="idx_ra_candidate_name_id", columns={"name_id"}),
- *          @ORM\Index(name="idxft_ra_candidate_email", columns={"email"}, flags={"FULLTEXT"}),
- *          @ORM\Index(name="idxft_ra_candidate_commonname", columns={"common_name"}, flags={"FULLTEXT"}),
- *          @ORM\Index(name="idx_ra_institution", columns={"ra_institution"})
- *      },
- *     uniqueConstraints={
- *          @ORM\UniqueConstraint(name="idx_ra_candidate_unique_identity_institution", columns={"identity_id", "ra_institution"})
- *     }
- * )
+ * Be aware that this entity is used for the RA Candidate view. This entity shouldn't be used to store any RA candidates.
+ * The IgnoreTablesListener is used to prevent schema changes when the Doctrine Schema tool is used.
+ *
+ * @ORM\Table(name="view_ra_candidate")
```

Comment:
This might need revision after venturing away from the view?

Reply:
Yes, that's true and should have been removed. I've directly removed the `IgnoreTablesListener` and the creation of the view in Deploy which could then also be removed.